client/css/widgets/holder.css

```diff
@@ -20,6 +20,18 @@
   top: 0;
 }
 
+.holder.showCardBacksToOtherPlayers > .foreign.card.widget {
+  display: block !important;
```

Comment:
Which rule is this trying to override, can you do it by ensuring the correct order or increasing specificity of this or decreasing speficity of the other instead of using `!important`? The more we use `!important` the harder it is for game authors to override, having to also specify `!important` like with transform on my perspective table.

Reply:
Should be this which already uses `!important`:

```CSS
body:not(.jsonEdit) .widget.foreign {
  display: none !important;
}
```

Reply:
Which was presumably used because it wasn't specific enough to override a `display: block` rule but that's going down a rabbit hole. Anyways we could always remove `!important` later.

Reply:
Shouldn't be _that_ hard to remove those two.

Reply:
Looks like `@layer` has pretty good support these days: https://developer.mozilla.org/en-US/docs/Web/CSS/@layer . We should switch to that so that author styles are used above vtt styles regardless of specificity. (Not suggesting in this patch)

Reply:
The `!important` is no longer needed because it no longer interacts with `.foreign`.